src/runes.c

```diff
@@ -317,6 +347,15 @@ void RuneTouch(void)
 	sound(other, CHAN_ITEM, "weapons/lock4.wav", 1, ATTN_NORM);
 	stuffcmd(other, "bf\n");
 	ent_remove(self);
+
+	TeamplayEventItemTaken(other, self);
+
+#ifdef BOT_SUPPORT
+	if (bots_enabled() && other->isBot)
```

Comment:
A bit confused, why do you check other->isBot, but generally not at other similar situations? If not checking that, this would be equivalent to `ItemTaken(self, other)`.